x/mint/keeper/keeper_test.go

```diff
@@ -26,6 +28,33 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.Setup()
 }
 
+// setupDeveloperVestingModuleAccountTest sets up test cases that utilize developer vesting
+// module account logic. It reverts some default logic added by suite.Setup()
+// Specifically, it removes the developer vesting module account
+// from account keeper if isDeveloperModuleAccountCreated is true.
+// Additionally, it initializes suite's Ctx with blockHeight
+func (suite *KeeperTestSuite) setupDeveloperVestingModuleAccountTest(blockHeight int64, isDeveloperModuleAccountCreated bool) {
+	suite.Setup()
+	// Reset height to the desired value since test suite setup initialized
+	// it to 1.
+	suite.Ctx = suite.Ctx.WithBlockHeader(tmproto.Header{Height: blockHeight})
+
+	// If developer module account is created, the suite.Setup() also correctly sets the offset
```

Comment:
```suggestion
	// If developer module account is not created, the suite.Setup() also correctly sets the offset
```

Maybe not? didn't fully understand what the comment here was indicating

Reply:
[Addressed](https://github.com/osmosis-labs/osmosis/pull/1872/commits/906f06f962c1b197130b21c346cbe74d143eaa92)